index.js

```diff
@@ -102,3 +102,27 @@ io.on('connection', socket => {
         console.log(`${currentUser} score: ${score}`);
     });
 });
+
+//Functions to perform fullscreen and chat toggle
+function goFullScreen() {
```

Comment:
Lastly, does it work on firefox? Please make sure it does, and please squash all your commits to a single commit :)